app/uk/gov/hmrc/alcoholdutycontactpreferences/testOnly/controllers/EventHubBounceController.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.alcoholdutycontactpreferences.testOnly.controllers
+
+import play.api.Logging
+import play.api.libs.json.JsValue
+import play.api.mvc.{Action, BaseController, ControllerComponents}
+import uk.gov.hmrc.alcoholdutycontactpreferences.testOnly.models.Event
+
+import javax.inject.Inject
+import scala.concurrent.Future
+
+class EventHubBounceController @Inject() (val controllerComponents: ControllerComponents)
+    extends BaseController
+    with Logging {
+
+  def handleBouncedEmail(): Action[JsValue] = Action.async(parse.json) { implicit request =>
```

Comment:
Do we need some kind of basic auth @simondrugan16 ?

Reply:
@simondrugan16  - This is the direct debit implementation of the very same journey we are looking to implement. 
https://github.com/hmrc/direct-debit-backend/blob/main/app/controllers/BouncedEmailController.scala#L26 

As long as we align with this I think we will be good.